setup.py

```diff
@@ -45,6 +45,7 @@
     "optuna",
     "sentencepiece",
     "datasets",
+    "timm",
```

Comment:
seems like a duplicate here. why there timm is added twice?

Reply:
This seems to be part of the tests dependencies, which is why I missed this. Would it be better to remove it from my example dependencies, or should I leave it due to `timm` not being included with the standard requirements?

Reply:
This is fine since it is for tests.